service/src/test/resources/ErlookupFailedWithOptionsInput.json

```diff
@@ -0,0 +1,53 @@
+{
+  "msgParams": {
+    "meta": {
+      "type": "EiffelTestCaseStartedEvent",
+      "version": "3.0.0",
+      "tags": [
+        ""
+      ],
+      "source": {
+        "domainId": "",
+        "host": "",
+        "name": "",
+        "uri": ""
+      }
+    }
+  },
+  "eventParams": {
+    "data": {
+      "executor": "",
+      "liveLogs": [],
+      "customData": []
+    },
+    "links": [
+      {
+        "type": "TEST_CASE_EXECUTION",
+            "%lookup%": {
+                "eventType": "EiffelArtifactCreatedEvent",
+                "properties": [{
+                    "data.identity": "pkg:maven/swdi.up/CXP102051_22@R21EK?type=xml&classifier=test"
+                }],
+                "options": {
+                    "failIfNoneFound": "true",
+                    "failIfMultipleFound": "false"
+                }
+            }
+      },
+      {
+            "type": "CAUSE",
+            "%lookup%": {
+                "eventType": "EiffelSourceChangeSubmittedEvent",
+                "properties": [{
+                    "data.gitIdentifier.commitId": "ad090b60a4aedc5161da9c035a49b14a319829b4",
+                    "data.gitIdentifier.repoUri": "https://github.com/johndoe/myPrivateRepo.git"
+                }],
+                "options": {
+                    "failIfNoneFound": "true",
+                    "falIfMultipleFoud": "true"
```

Comment:
Has this passed your tests? It should say 'failIfMultipleFou**n**d'

Reply:
In this test I given a wrong name 'failIfMultipleFoud' for testing the scenario if the parameter names are wrong it should throws an error message with status code '422'.

Reply:
Ah. Ok. My bad :)